package.json

```diff
@@ -15,15 +15,17 @@
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.3",
     "@react-native-masked-view/masked-view": "^0.2.8",
-    "@react-navigation/bottom-tabs": "^6.0.9",
+    "@react-navigation/bottom-tabs": "^6.5.7",
     "@react-navigation/drawer": "^6.1.8",
     "@react-navigation/native": "^6.0.6",
+    "@react-navigation/native-stack": "^6.9.12",
     "@react-navigation/stack": "^6.0.11",
     "axios": "^0.26.0",
     "eslint-plugin-prettier": "^4.1.0",
     "react": "17.0.2",
     "react-native": "0.66.4",
     "react-native-app-auth": "^6.4.2",
+    "react-native-date-picker": "^4.2.13",
     "react-native-datepicker": "^1.7.2",
```

Comment:
@JAYATEJA04 , I see that for date picker we are using two libraries here. Could you please confer with @shreya-mishra once and decide on one library which is better suited for us?
Additionally, there is a component called DatePicker at /components/createGoalsForm. Please take care of that as well.

Reply:
Has this been resolved yet?

Reply:
For Shreya and me to review could you please tell, why you decided to go with this library when one already existed and created a new component which also existed?

Reply:
I'm not aware about this library and component already existed, though both the library's functionalities are the same but both are two different libraries,
react-native-date-picker: https://www.npmjs.com/package/react-native-date-picker
react-native-datepicker: https://www.npmjs.com/package/react-native-datepicker
I created this component without knowledge of its existence, I didn't see its use case in the app when I started working on it.
I shall have to check it with @shreya-mishra

Reply:
@JAYATEJA04 please check before installing any libraries. And also both works the same. Try to use the same library or if there is any difference of performance or anything let us know the same.

Reply:
Can I uninstall the library which was previously installed? That seems to be better because I'm familiar with the latest one and I have implemented the same in forms.

Reply:
Hello, @shreya-mishra I have noticed that there exists DatePicker component at `/components/createGoalsForm` and not only this, there also exists createGoalsForm component which is similar to mine. Should I delete the createGoalsForm component and uninstall the DatePicker library too?

Reply:
we are going ahead with yours for now :) dont delete yours but use the same library which is already there

Reply:
Ok noted.

Reply:
![ss1](https://github.com/Real-Dev-Squad/mobile-app/assets/57326919/d8d1f638-23e2-4974-b45b-8f60bc037f03)

Hello, @shreya-mishra I have tried replacing the date picker component from the one I have installed to the one which already existed, when I run the app I have noticed that this has been removed. Should I go ahead with the one which I have created and uninstall the previous one?

Reply:
So I will be going ahead with the one which I have used and uninstalling the library which already existed.

Reply:
If  I delete the existing library, I should also delete those files that use that library. Should I have to delete those files too? Or should I let both libraries remain?

Reply:
deleting the files goes to a different ticket, please finish this task for now and then you may create a new ticket called deleting the deprecated library and assign it to yourself and start working on that.

Reply:
Then I will go ahead with keeping both the libraries and I will be using the library which I have installed in goals form.

Reply:
This is resolved.